packages/odyssey/src/scss/abstracts/_functions.scss

```diff
@@ -12,26 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Rem-based modular scale for type
- */
-
-@function ms($step) {
-  $value: 1rem;
-
-  @if $step > 0 {
-    @for $i from 1 through $step {
-      $value: ($value * $scale-ratio);
-    }
-  }
-
-  @if $step < 0 {
-    @for $i from $step through -1 {
-      $value: ($value / $scale-ratio);
-    }
+@function rem-to-px($value) {
```

Comment:
I haven't docced these before, but can we add some comments around expected param format and output unit?